scripts/calibrateArmDac.sh

```diff
@@ -1,26 +1,51 @@
-#!/bin/zsh
+#!/bin/bash
 # Usage:
-#   executeScurveProgram.sh <DetName> <cardName> <link> <vfatmask> <Comma Separated List of CFG_THR_ARM_DAC Values>
+#   calibrateArmDac.sh <DetName> <cardName> <link> <vfatmask> <Comma Separated List of CFG_THR_ARM_DAC Values>
 
 helpstring="Usage:
-    ./executeScurveProgram.sh [DetName] [cardName] [link] [vfatmask] [comma separated list of CFG_THR_ARM_DAC Values]
+    ./calibrateArmDac.sh [DetName] [cardName] [shelf]  [slot] [link] [vfatmask] [comma separated list of CFG_THR_ARM_DAC Values]
+    ./calibrateArmDac.sh -r [PATH] [DetName] [cardName] [shelf] [slot] [link] [vfatmask] [comma separated list of CFG_THR_ARM_DAC Values]
         
     For each CFG_THR_ARM_DAC value specified will create a scandate directory, configure, print the configuration, and take an scurve
 
         DetName: Serial Number of Detector, this should be a subfolder under DATA_PATH variable
         cardName: CTP7 network alias or ip address
         link: OH number on the CTP7
+        shelf: Shelf number
+        slot: Slot number 
         vfatmask: 24-bit mask where a 1 in the n^th bit implies the n^th VFAT shall be excluded
+
+    If -r [PATH] is provided, the directory [PATH] will be used instead of creating a new scandate directory and ARM DAC values for which a scurve log file already exists in that directory will be skipped
 "
 
+ISFILE=0;
+OPTIND=1
+while getopts "r:h" opts
+do
+    case $opts in
+        r)
+            RESUMEDIR=$OPTARG;;
+        h)
+            echo ${helpstring};;
+        \?)
+            echo ${helpstring};;
+        [?])
+            echo ${helpstring};;
+    esac
+done
+shift $((OPTIND-1))
+unset OPTIND
```

Comment:
if you're going to implement `getopts` might as well make an option for each of the required inputs.

Reply:
and update usage example and `helpstring` accordingly.  See for example `setup_gemdaq.sh`

Reply:
done